### ISSUE TITLE
Same Output for each image

### ISSUE THREAD

**Issue by mohkoh19** [mlmed/torchxrayvision#89]
I've tried to use the pertained models on Images from MIMIC-CXR. For each model I get the same probabilities for each image, even if they are very different from each other. 

For example: 
<img width="222" alt="image" src="https://user-images.githubusercontent.com/16286447/155973431-0d869306-0f53-4c98-a15b-352a6b70645a.png">
<img width="222" alt="image" src="https://user-images.githubusercontent.com/16286447/155973456-85633068-e5d2-446c-ba45-3d02968b4185.png">

Both images have been center cropped and resized to 224x224 resolution. I used the following code to get outputs from the model:

```
model = xrv.models.DenseNet(weights="densenet121-res224-all")

with torch.no_grad(): 
    out0 = model(inputs['image'][0].unsqueeze(0))
    out1 = model(inputs['image'][1].unsqueeze(0))
 ```

Both `out0` and `out1` equal 
`tensor([[0.5995, 0.5762, 0.5281, 0.5486, 0.5477, 0.5251, 0.5390, 0.6766, 0.5525,
         0.5918, 0.6122, 0.5254, 0.5949, 0.3159, 0.5500, 0.5202, 0.6688, 0.6592]])`

The same problem occurs with other pertained models like the one for MIMIC-CXR as well, only with different probabilities, but still the same ones for each respective image.

**Comment by ieee8023** [comment]
I just tried with two random images from MIMIC and the outputs change with [scripts/process_image.py](https://github.com/mlmed/torchxrayvision/blob/master/scripts/process_image.py):
```
$ python3 process_image.py mimic/s50752076/90c22043-e189c0ff-6fb7432e-fe2dbddd-6e254412.jpg 
{'preds': {'Atelectasis': 0.8238672,
           'Cardiomegaly': 0.7937068,
           'Consolidation': 0.6639328,
           'Edema': 0.64718884,
           'Effusion': 0.8994624,
           'Emphysema': 0.34765577,
           'Enlarged Cardiomediastinum': 0.6454991,
           'Fibrosis': 0.3462754,
           'Fracture': 0.54863507,
           'Hernia': 0.10696117,
           'Infiltration': 0.60848004,
           'Lung Lesion': 0.5194453,
           'Lung Opacity': 0.7979231,
           'Mass': 0.51675844,
           'Nodule': 0.43947062,
           'Pleural_Thickening': 0.39090982,
           'Pneumonia': 0.5910077,
           'Pneumothorax': 0.52356243}}
BFLY-C02G10AMML7H-MBP:scripts ieee8023$ python3 process_image.py mimic/s50752076/a5b86009-73d89547-016407ee-f7d9a239-9bbc4465.jpg 
{'preds': {'Atelectasis': 0.78038526,
           'Cardiomegaly': 0.72883594,
           'Consolidation': 0.67473954,
           'Edema': 0.64642406,
           'Effusion': 0.82089716,
           'Emphysema': 0.51148427,
           'Enlarged Cardiomediastinum': 0.61464787,
           'Fibrosis': 0.5282905,
           'Fracture': 0.55740416,
           'Hernia': 0.10064988,
           'Infiltration': 0.57235926,
           'Lung Lesion': 0.5776483,
           'Lung Opacity': 0.77271074,
           'Mass': 0.54339457,
           'Nodule': 0.51708037,
           'Pleural_Thickening': 0.52615666,
           'Pneumonia': 0.55208266,
           'Pneumothorax': 0.5493742}}
```

Potentially you are not scaling the images to between [-1024, 1024] ?

**Comment by mohkoh19** [comment]
Thank you for your reply. I used the `ToTensor()` transform from torchvision, which scales between [0,1]. I changed it to [0,255] and I can confirm your output. Should've read the paper...

**Comment by ieee8023** [comment]
I think we should probably issue a warning if the input images are not scaled correctly (by ensuring there are values outside the range [-1,255]). I've mainly just been afraid of slowing down performance but I think we can only check on the first batch and it won't hurt performance for batch processing.

**Comment by ieee8023** [comment]
> I changed the scaling to [0,255]

@mohkoh19 do you mean to ran the code `xrv.datasets.normalize(img, 255)` ? The correct scaling range is [-1024, 1024]. The 255 in that function call indicates it will take an 8-bit image (with values 0-255) and scale it up.

**Comment by mohkoh19** [comment]
When I answered I just removed the [0, 1] scaling at the end of the pre-processing leading to [0, 255] scaling and could see differences in output already. I rescaled to [-1024, 1024] later using the `xrv.datasets.normalize(img, 255)` function and it works fine, thank you.

**Comment by ieee8023** [comment]
The feature discussed is now released in https://github.com/mlmed/torchxrayvision/releases/tag/0.0.37 !
